Integrations/AttackIQFireDrill/AttackIQFireDrill.py

```diff
@@ -0,0 +1,475 @@
+
+
+def build_test_results_hr(test_results, test_id):
+    """
+    Creates test results human readable
+    Args:
+        test_results (list): Results of the test (after being transformed)
+        test_id (str): ID of the test
+
+    Returns: Human readable of test results
+    """
+    keys = ['Assessment Name', 'Scenario Name', 'Hostname', 'Asset IP', 'Job State', 'Modified', 'Outcome']
+    test_results_mod = []
+    for t_res in test_results:
+        assessment = t_res.get('Assessment')
+        asset = t_res.get('Asset')
+        scenario = t_res.get('Scenario')
+        hr_items = {
+            keys[0]: assessment.get('Name'),
+            keys[1]: scenario.get('Name'),
+            keys[2]: asset.get('Hostname'),
+            keys[3]: asset.get('Ipv4Address'),
+            keys[4]: demisto.get(t_res, 'JobState.Name'),
+            keys[5]: t_res.get('Modified'),
+            keys[6]: demisto.get(t_res, 'Outcome.Name')
+        }
+        test_results_mod.append(hr_items)
+    return tableToMarkdown(f'Test Results for {test_id}', test_results_mod, keys)
+
+
+def get_test_results_command(args=demisto.args()):
+    """ Implements attackiq-get-test-results command
+    """
+    test_id = args.get('test_id')
+    params = {
+        'page': args.get('page', '1'),
+        'test_id': test_id,
+        'show_last_result': args.get('show_last_result') == 'True'
+    }
+    try:
+        raw_test_res = http_request('GET', '/v1/results', params=params)
+        test_res = build_transformed_dict(raw_test_res['results'], TEST_RESULT_TRANS)
+        hr = build_test_results_hr(test_res, test_id)
+        return_outputs(hr, {'AttackIQTestResult(val.Id === obj.Id)': test_res}, raw_test_res)
+    except HTTPError as e:
+        return_error(create_invalid_id_err_msg(str(e), ['500']))
+
+
+def get_assessments(page='1', assessment_id=None):
+    """
+    Fetches assessments from attackIQ
+    Args:
+        page (str or int): Page number to fetch
+        assessment_id (str): (Optional) If provided will fetch only the assessment with matching ID
+
+    Returns: Assessments from attackIQ
+    """
+    if assessment_id:
+        return http_request('GET', f'/v1/assessments/{assessment_id}')
+    return http_request('GET', '/v1/assessments', params={'page': page})
+
+
+def list_assessments_command():
+    """ Implements attackiq-list-assessments command
+    """
+    page = demisto.args().get('page_number')
+    try:
+        page_size = int(demisto.args().get('page_size', DEFAULT_PAGE_SIZE))
+        page = int(page)
+    except (ValueError, TypeError):
+        page_size = DEFAULT_PAGE_SIZE
+    raw_assessments = get_assessments(page=page)
+    assessments_res = build_transformed_dict(raw_assessments.get('results'), ASSESSMENTS_TRANS)
+    ass_cnt = raw_assessments.get('count')
+    total_pages = (ass_cnt // page_size) + 1
+    try:
+        remaining_pages = total_pages - page
+    except (ValueError, TypeError):
+        remaining_pages = None
```

Comment:
if the user did not insert a value to `page_number ` the result should not be that the remaining pages is none.
If the user ran the command for the first time it Is natural that the page number will not be specified, but we still expect to return the remaining pages. 

I think it is better to put zero(or 1) in the default, and return an error if there is a non neural number there.
the same for `page_size`

Reply:
replaced implementation, so now it'll always be `integers`, so the math operation will never fail